chutney/server/src/test/resources/sso/sso-oidc.mjs

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import express from 'express';
+import { Provider } from 'oidc-provider';
+
+const oidc = new Provider('http://localhost:3000', {
```

Comment:
Can we use here some environment variables for :
* listenning port
* redirect url host + port

Maybe in the spring yaml file, it could be good too ?